### PR TITLE
Get all items

### PR DESCRIPTION
**Pull request by Jonathan-Atkins**
updated get_items_all with error message (using serializer)

### CODE REVIEW

File: spec/rails_helper.rb
Comment (wally-yawn) [inline spec/rails_helper.rb:1]: We still need all the simplcov and shouldmatchers here.

**Review by wally-yawn — CHANGES_REQUESTED** [review]

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -14,7 +14,6 @@
 #
 # See https://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 RSpec.configure do |config|
-  config.formatter = :documentation
```
Comment (wally-yawn) [inline spec/spec_helper.rb:17]: we still need this formatter

**Review by wally-yawn — APPROVED** [review]
Looks good now, approved

File: app/controllers/api/v1/items_controller.rb
Comment (James-Cochran) [inline app/controllers/api/v1/items_controller.rb:1]: Nice use of rescue_from

File: app/serializers/error_serializer.rb
Comment (James-Cochran) [inline app/serializers/error_serializer.rb:1]: I like the creation of the error serializer.  Lets connect on refactoring everything else to use it.

**Review by James-Cochran — APPROVED** [review]

**Pull request merged**